### PR TITLE
added complementary color section

### PR DESCRIPTION
**Pull request by FernandisNewton**
issue #22

### CODE REVIEW

**Comment by FernandisNewton** [comment]
Screenshot 2022-10-09 at 12 48 02 PM

**Comment by SscSPs** [comment]
Looks great, but why have you reduced it to a single row? It originally had saturation and luminosity spectrum

File: src/ColorGrid.js [hunk 1]
```diff
@@ -14,56 +14,60 @@ const ColorGrid = ({ hueValue, saturationCount, luminosityCount }) => {
...
-          })
+        // create saturationCount div
+        <div style={{ display: "flex", flexDirection: "row" }}>
+          {
+            // create luminosityCount divs
+            Array(luminosityCount - 1)
+              .fill(0)
+              .map((_, j) => {
+                var hueLocal = hueValue;
+                var saturationLocal = j * saturationFraction;
```
Comment (SscSPs) [inline src/ColorGrid.js]: Looks like a previous PR merged a typo, this needs to be i and not j,
please fix this bug and see if your changes work good still, 
I was thinking more like 2 grids with the hue to be different

**Review by SscSPs — CHANGES_REQUESTED** [review]

File: src/ColorGrid.js (hunk 1 above)
Comment (FernandisNewton) [reply src/ColorGrid.js]: Sure sure 👍🏼

**Comment by FernandisNewton** [comment]
> 
### I reduced to one row because the same row was repeating multiple times.
Screenshot 2022-10-09 at 12 59 01 PM

File: src/ColorGrid.js [hunk 2]
```diff
@@ -14,56 +14,60 @@ const ColorGrid = ({ hueValue, saturationCount, luminosityCount }) => {
...
+        // create saturationCount div
+        <div style={{ display: "flex", flexDirection: "row" }}>
+          {
+            // create luminosityCount divs
+            Array(luminosityCount - 1)
+              .fill(0)
+              .map((_, i) => {
+                var hueLocal = hueValue;
+                var saturationLocal = i * saturationFraction;
+                var luminosityLocal = i * luminosityFraction;
```
Comment (SscSPs) [inline src/ColorGrid.js:30]: You changed both, one needs to be i and other needs to be j

Comment (SscSPs) [reply src/ColorGrid.js:30]: And needs to be grid, not a single row

Comment (FernandisNewton) [reply src/ColorGrid.js:30]: Got it 👍🏼 
Will fix that.

**Comment by SscSPs** [comment]
@FernandisNewton Hey, I've fixed the bug that caused the single palette being shown across the grid, 
Please rebase your changes on the master now and it should be great!

**Comment by SscSPs** [comment]
Hey @FernandisNewton, let me know if you are still interested in working on this, otherwise, we can let someone else work on this :)

**Comment by SscSPs** [comment]
no response from you, so closing this PR

**Pull request closed without merging**